tox.ini

```
@@ -79,21 +72,20 @@ deps =
     {[testenv:type_check]deps}
     {[testenv:coverage]deps}
     {[testenv:spell_check]deps}
-    -r dev_requirements/requirements-formatting.txt
+    .[formatting]
     pip-tools
     pre-commit
 commands =
     python -m pip install --upgrade pip
     pip install --upgrade virtualenv
-    pip-compile pyproject.toml
-    pip-compile --extra aiostream -o dev_requirements/requirements-extra-aiostream.txt pyproject.toml
+    pip-compile --extra aiostream pyproject.toml
```

Hier solltest du einen Output-Pfad angeben. `seviper` hat zwar an sich keine Dependencies, aber 1. ist das nicht in Stein gemeißelt und 2. liest sich `requirements.txt` nicht wie eine Datei, die Dependencies für `seviper[aiostream]` enthält.

```suggestion
    pip-compile --extra aiostream -o requirements-extra-aiostream.txt pyproject.toml
```